### ISSUE TITLE
[Laravel] How to use _all mapping?

### ISSUE THREAD

**Issue by terion-name** [basemkhirat/elasticsearch#20]
It seems that mappings from config are passed directly to `properties` of Elastic mapper. But how to set `_all` map? that is on higher level of map document?

**Comment by basemkhirat** [comment]
Fixed in v0.9.9. just update your version
```php

'indices' => [
	'my_index_1' => [
	    "aliases" => [
	        "my_index"
	    ],
	    'settings' => [
	        "number_of_shards" => 1,
	        "number_of_replicas" => 0,
	    ],
	    'mappings' => [
	          "_all" => [ "enabled" => true ],
                  "properties" => [
                      'posts' => [
                           'title' => [
                               'type' => 'string'
                           ]
                      ]
	          ]
	    ]
	]
]

```

**Comment by terion-name** [comment]
thank you
